api/end_device.proto

```diff
@@ -270,6 +270,8 @@ message MACSettings {
   // Maximum EIRP (dBm).
   // If unset, the default value from regional parameters specification will be used.
   DeviceEIRPValue desired_max_eirp = 30;
+  // The minimum duration passed before a network-initiated(e.g. Class B or C) downlink following an arbitrary downlink.
+  google.protobuf.Duration network_initiated_downlink_interval = 31 [(gogoproto.stdduration) = true];
```

Comment:
I think we use `class_b_c` in other places; that would be instead of `network_initiated`. I know it's not consistent with NS terminology, but this is API so that should stay consistent with API, that is more important than introducing NS terms here.